molecule/check_facts/tasks/check.yml

```diff
@@ -0,0 +1,11 @@
+---
+
+- fail:
+    msg: "Task 'check_task' wasn't started"
```

Comment:
Prepare wasn't started?

Reply:
This task wasn't started, prepare will be started in any case)